src/codec_svt.c

```diff
@@ -125,14 +132,20 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         svt_config->is_16bit_pipeline = image->depth > 8;
 #endif
 
+#if !SVT_AV1_CHECK_VERSION(3, 0, 0)
         // Follow comment in svt header: set if input is HDR10 BT2020 using SMPTE ST2084 (PQ).
         svt_config->high_dynamic_range_input = (image->depth == 10 && image->colorPrimaries == AVIF_COLOR_PRIMARIES_BT2020 &&
                                                 image->transferCharacteristics == AVIF_TRANSFER_CHARACTERISTICS_SMPTE2084 &&
                                                 image->matrixCoefficients == AVIF_MATRIX_COEFFICIENTS_BT2020_NCL);
+#endif
```

Comment:
It just occurred to me that since `high_dynamic_range_input` seems to be never used, we might as well just delete it. We can submit this version first and confirm with the SVT-AV1 team that `high_dynamic_range_input` is never used.

Reply:
It got used at some point right? Many eons ago.

Reply:
@1480c1 Christopher: Does `svt_config->high_dynamic_range_input` have an effect in any SVT-AV1 release? If not, then we will just delete this assignment statement rather than keep it under a `#if !SVT_AV1_CHECK_VERSION(3, 0, 0)` test.

Reply:
As far as I know, it has never had an effect since SVT-AV1's public release.

Reply:
done